collector/nginx_plus.go

```diff
@@ -359,22 +418,34 @@ func (c *NginxPlusCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 
 	for name, zone := range stats.StreamServerZones {
+		labelValues := []string{name}
+		varLabelValues := c.getServerZoneLabelValues(name)
+
+		if c.variableLabelNames.ServerZoneVariableLabelNames != nil && len(varLabelValues) != len(c.variableLabelNames.ServerZoneVariableLabelNames) {
+			log.Printf("wrong number of labels for tcp zone %v. For labels %v, got values: %v. Empty labels will be used instead",
```

Comment:
Better to use stream server zone I think.

The zone could have upstreams of protocol udp, tcp or (as far as I know) a mix.

```suggestion
			log.Printf("wrong number of labels for stream server zone %v. For labels %v, got values: %v. Empty labels will be used instead",
```